README.md

```diff
@@ -27,118 +25,28 @@ other information, for example) is expected to be stored outside of the graph
 structure itself. Such data lends itself to storage in more traditional and
 better-optimized mechanisms.
 
-Additional functionality may be found in the companion package [LightGraphsExtras.jl](https://github.com/JuliaGraphs/LightGraphsExtras.jl).
+Additional functionality may be found in a number of companion packages, including:
```

Comment:
Maybe this should say something like:

> LightGraphs.jl is the central package of the JuliaGraphs ecosystem. Additional functionality such as, advanced IO and file formats, weighted graphs, property graphs, and optimization related functions can be found in the following packages.

And then sort the list in this order. I think this order is most likely to be clicked to least likely.

Reply:
✓